libs/react-helpers/src/lib/format/date.ts

```diff
@@ -31,6 +31,13 @@ export const getDateTimeFormat = once(
     })
 );
 
+export const dateValueFormatter = ({ value }: { value?: string | null }) => {
```

Comment:
suggestion (non blocking):  I'm not sure we really need extra formatter. Just use getDateTimeFormat().format()?

Reply:
updated.